src/lib.rs

```diff
@@ -1117,14 +1130,27 @@ impl<T: Counter> Histogram<T> {
 
     /// Computes the matching histogram value for the given histogram bin.
     pub fn value_for(&self, index: usize) -> u64 {
+        // TODO what to do about indexes that are beyond what can be represented by this histogram?
+        // Pretty easy to end up shifting off the high end of a u64, e.g. asking for a big index
+        // when sigfigs is high.
```

Comment:
Maybe the way to go is to add auto-unwrapping shorthands (like `AddAssign`) that people can use if they are willing to have the code panic. I agree with you that it's better to have the *option* of getting a `Result` instead of a panic, so let's make that change.